pyqt_sql_demo/widgets/text.py

```diff
@@ -1,8 +1,13 @@
 CONNECTION_TAB_DEFAULT_TITLE = "Untitled"
-CONNECTION_STRING_SUPPORTED_DB_NAMES = ["SQLite"]
+CONNECTION_STRING_SUPPORTED_DB_NAMES = ["SQLite", "MySQL"]
 CONNECTION_STRING_PLACEHOLDER = "Enter..."
-CONNECTION_STRING_DEFAULT = "demo.db"
-QUERY_EDITOR_DEFAULT_TEXT = "SELECT name FROM sqlite_master WHERE type='table'"
+
+CONNECTION_STRING_SQL_LITE_DEFAULT = "demo.db"
+CONNECTION_STRING_MYSQL_DEFAULT = "username=root;host=localhost;port=3306;dbname=mysql"
```

Comment:
Naming inconsistencies:

CONNECTION_STRING_MYSQL_DEFAULT - database name is not the last word of the variable name
QUERY_EDITOR_DEFAULT_TEXT_MYSQL - database name is the last word

I'd just put it as a last word everywhere